### PR TITLE
Fix aerogear smoke tests. Move aerogear.reddeer classes to correct pa…

### PR DESCRIPTION
**Pull request by rawagner**
…ckages

### CODE REVIEW

File: plugins/org.jboss.tools.aerogear.reddeer/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -18,9 +18,12 @@ Bundle-ActivationPolicy: lazy
...
 Eclipse-BundleShape: jar
 Bundle-Vendor: JBoss by Red Hat
-Export-Package: org.jboss.tools.aerogear.reddeer.ui.config,
- org.jboss.tools.aerogear.reddeer.ui.preferences,
- org.jboss.tools.aerogear.reddeer.ui.properties,
- org.jboss.tools.aerogear.reddeer.ui.wizard,
- org.jboss.tools.aerogear.reddeer.ui.wizard.export,
- org.jboss.tools.aerogear.reddeer.cordovasim
+Export-Package: org.jboss.tools.aerogear.reddeer.cordovasim,
+ org.jboss.tools.aerogear.reddeer.thym.nadroid.ui.preferences,
```
Comment (psrna) [inline plugins/org.jboss.tools.aerogear.reddeer/META-INF/MANIFEST.MF]: typo error I guess, nadroid -> android

File: tests/org.jboss.tools.aerogear.ui.bot.test/AerogearAllBotTests.reddeer.launch [hunk 2]
```diff
@@ -20,7 +20,7 @@
 <listEntry value="1"/>
 </listAttribute>
 <mapAttribute key="org.eclipse.debug.core.environmentVariables">
-<mapEntry key="DISPLAY" value=":1"/>
+<mapEntry key="DISPLAY" value=":1.0"/>
```
Comment (psrna) [inline tests/org.jboss.tools.aerogear.ui.bot.test/AerogearAllBotTests.reddeer.launch]: Why 1.0?

File: tests/org.jboss.tools.aerogear.ui.bot.test/src/org/jboss/tools/aerogear/ui/bot/test/app/DisplayJavaScriptErrors.java [hunk 3]
```diff
@@ -11,60 +11,56 @@
...
+		final String errorVariable = "ERROR_VARIABLE";
+		jsString = jsString.replaceFirst("app\\.receivedEvent\\('deviceready'\\);",
+				"app.receivedEvent(\'deviceready\');" + "\nconsole.log(\"" + logMessage + "\");\n" + errorVariable);
+		new DefaultStyledText().setText(jsString);
+		jsEditor.save();
+		runCordovaSim(CORDOVA_PROJECT_NAME);
+		CordovaSimLauncher.stopCordovasim();
+		String consoleText = new ConsoleView().getConsoleText();
+		String textToContain = "LOG: " + logMessage;
+		// failshere
```
Comment (rawagner) [inline tests/org.jboss.tools.aerogear.ui.bot.test/src/org/jboss/tools/aerogear/ui/bot/test/app/DisplayJavaScriptErrors.java]: remove comment

File: tests/org.jboss.tools.aerogear.ui.bot.test/AerogearAllBotTests.reddeer.launch (hunk 2 above)
Comment (rawagner) [inline tests/org.jboss.tools.aerogear.ui.bot.test/AerogearAllBotTests.reddeer.launch]: Doesnt matter, thats just what I use, removed

File: plugins/org.jboss.tools.aerogear.reddeer/META-INF/MANIFEST.MF (hunk 1 above)
Comment (rawagner) [inline plugins/org.jboss.tools.aerogear.reddeer/META-INF/MANIFEST.MF]: fixed

File: tests/org.jboss.tools.aerogear.ui.bot.test/src/org/jboss/tools/aerogear/ui/bot/test/app/DisplayJavaScriptErrors.java (hunk 3 above)
Comment (rawagner) [inline tests/org.jboss.tools.aerogear.ui.bot.test/src/org/jboss/tools/aerogear/ui/bot/test/app/DisplayJavaScriptErrors.java]: removed

**Pull request merged**